bats/fb-content-katello.bats

```diff
@@ -10,91 +10,97 @@ setup() {
   tSetOSVersion
 }
 
+@test "Create an Empty Organization" {
+  run hammer -u admin -p changeme organization info --name "Empty Organization"
```

Comment:
do you mind adding a comment here to explain why this org is being created? (so that we are assured a 2nd org exists when we try to delete Test Org)